internal/core/cobra_builder.go

```diff
@@ -204,15 +209,19 @@ DEPRECATED ARGS:
 {{- range $_, $group := orderGroups (getCommandsGroups .Commands) }}
 
 {{ $group.Title }} COMMANDS:
-{{- range $_, $command := orderCommands $.Commands }}
-{{- if $command.IsAvailableCommand }}
+  {{- range $_, $command := orderCommands $.Commands }}
+  {{- if or $command.IsAvailableCommand $command.Deprecated }}
   {{- if or ($command.ContainsGroup $group.ID) (and (eq $group.ID "utility") (eq $command.Name "help")) }}
-  {{ rpad $command.Name .NamePadding }} {{ if $command.Short }}{{ $command.Short }}{{end}}
-  {{- end }}
+  {{ rpad $command.Name .NamePadding }}
+  {{- if $command.Deprecated }}{{ if $command.Short }}{{ $command.Short }} (Deprecated){{ end }}
```

Comment:
This should fix alignement
```suggestion
  {{- if $command.Deprecated }} {{ if $command.Short }}{{ $command.Short }} (Deprecated){{ end }}
```